### PR TITLE
[9.0][FIX] partner_financial_risk: Invoice refunds add risk.

### PR DESCRIPTION
**Pull request by carlosdauden**
FIX:
 - Refunds adds partner risk instead of reduce.
 - If date_maturity changes to previous today date "never" compute partner risk.

@Tecnativa

### CODE REVIEW

File: partner_financial_risk/models/account_move.py
Comment (pedrobaeza) [inline partner_financial_risk/models/account_move.py]: Remove `()`

**Review by pedrobaeza — CHANGES_REQUESTED** [review]
Change module version number

**Comment by pedrobaeza** [comment]
There's a conflict.

File: partner_financial_risk/models/res_company.py
Comment (pedrobaeza) [inline partner_financial_risk/models/res_company.py]: Isn't there an specific method for applying changes in this screen?

Comment (carlosdauden) [reply partner_financial_risk/models/res_company.py]: It can be written from settings or from company, although it is not currently in the company view.

Comment (pedrobaeza) [reply partner_financial_risk/models/res_company.py]: OK, let's do it this way then.

**Review by pedrobaeza — APPROVED** [review]

**Comment by pedrobaeza** [comment]
cc @cubells @luismontalba

File: partner_financial_risk/models/account_move.py
Comment (cubells) [inline partner_financial_risk/models/account_move.py]: You should change `© symbol` by `Copyright`

**Review by cubells — APPROVED** [review]
:+1: 

Minor changes

LGTM. 

Code review and tests

**Review by luismontalba — APPROVED** [review]

**Comment by rafaelbn** [comment]
This PR looks ready to merge. Thanks @carlosdauden !

**Comment by carlosdauden** [comment]
Squash before merge.

**Comment by rafaelbn** [comment]
This fixes must be applied in the migration to v10 https://github.com/OCA/partner-contact/pull/470

**Comment by carlosdauden** [comment]
I could have rebased the last 6 commits...

**Comment by pedrobaeza** [comment]
@chienandalu please put this in v10

**Comment by pedrobaeza** [comment]
@rafaelbn please don't merge without squashing

**Pull request merged**
